.github/workflows/incluster-comp-pr-merged.yaml

```
@@ -117,50 +117,73 @@ on:
               name: Installing go
               with:
                   go-version: "${{ inputs.GO_VERSION }}"
 
-            - name: Set up QEMU
-              uses: docker/setup-qemu-action@v3
+            - name: Run unit test
+              run: go test -v ./...
+
+    build:
+        needs: [setup, unit-test]
+        if: ${{ matrix.arch == 'amd64' || contains(inputs.BUILD_PLATFORM, 'arm64') }}
```

actionlint (1.7.12)

[error] context "matrix" is not allowed here. available contexts are "github", "inputs", "needs", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)